src/test/java/net/nuagenetworks/bambou/RestCertificateTest.java

```diff
@@ -0,0 +1,244 @@
+	+ "DDAKBgNVBAsMA0tJQTELMAkGA1UEAwwCTFAxHTAbBgkqhkiG9w0BCQEWDm5vbmVA\n"
+	+ "eWFob28uY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAtkci4PS7\n"
+	+ "xjd9j4QQiLKe86n9wCAQAVlarEdKb+1Uv2uOJKdAAnhhclKr52rjvlBu0ZXeTVZ0\n"
+	+ "HXasIijFqA0+huwNNXk99CmTvIkqUsIow8x5MSTp3gU9X6uW4QtbI/C5kiNCpWHm\n"
+	+ "Xr3H3mgz0kzYi4vCYxAf7fPM9qtNyo92feiDKLUhYNtfja/cM3BWzm/vpEwmz9v7\n"
+	+ "Mp9204GbQSontb4oDDJuZPN3i+XGHlRerlREg+ScjyQH0u1UjujmOervNeHCVaCs\n"
+	+ "/emamOzv/6HY9wAB3AsWxKDX5tbrAjHgm61cCC3UESoHDApyzE1OcaNu/AsLo4iq\n"
+	+ "NspCZTk9+TPXgQIDAQABo1AwTjAdBgNVHQ4EFgQU8v25fJq45gkYrS75spZ01W47\n"
+	+ "5oMwHwYDVR0jBBgwFoAU8v25fJq45gkYrS75spZ01W475oMwDAYDVR0TBAUwAwEB\n"
+	+ "/zANBgkqhkiG9w0BAQsFAAOCAQEAQ48yKSwoPEmc7kGvrRIqdnBuZMAaGBO72xOZ\n"
+	+ "S4jQl2OPMYxGRWW7Km21L9vocDim9GYSmaKNm8uGv/E4XWpgyY609Vai7RdA1qEJ\n"
+	+ "JuW2KsmraxstfdUiXwYDPS4AC3hLtnW473mxyfa0BdzJMeO+/ZYrDppJhk1kvbHa\n"
+	+ "usgBI7vgzTbckq9+vNoGdrvzHSHNbxW5xKMO7thqgoUxijShSojJ/AXARtB6hBbG\n"
+	+ "EwOuXHe7Jop0nLzQe46jPgn9q+Fe2gYv26EskUXZAAoDKMZTYgjnfuXinZMKpQ7N\n"
+	+ "vUerIa96ds41U3YszcracJrHcz+qVtyWcyjAnGg6YIPIeSo/cw==\n"
+	+ "-----END CERTIFICATE-----\n";
+
+	public static final String BAD_TESTABLE_CERTIFICATE =
+	"-----BEGIN CERTIFICATE-----\n"
+//	+ "MIIDsTCCApmgAwIBAgIJAOx9LprAoRmEMA0GCSqGSIb3DQEBCwUAMG4xCzAJBgNV\n"
+//	+ "BAYTAlVTMQswCQYDVQQIDAJOSjELMAkGA1UEBwwCU0IxCzAJBgNVBAoMAk5PMQww\n"
+	+ "CgYDVQQLDANLSUExCzAJBgNVBAMMAkxQMR0wGwYJKoZIhvcNAQkBFg5ub25lQHlh\n"
+	+ "aG9vLmNvbTAgFw0xNzA1MDIxNjI3MTZaGA80NzU1MDMyOTE2MjcxNlowbjELMAkG\n"
+	+ "A1UEBhMCVVMxCzAJBgNVBAgMAk5KMQswCQYDVQQHDAJTQjELMAkGA1UECgwCTk8x\n"
+	+ "DDAKBgNVBAsMA0tJQTELMAkGA1UEAwwCTFAxHTAbBgkqhkiG9w0BCQEWDm5vbmVA\n"
+	+ "eWFob28uY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAtkci4PS7\n"
+	+ "xjd9j4QQiLKe86n9wCAQAVlarEdKb+1Uv2uOJKdAAnhhclKr52rjvlBu0ZXeTVZ0\n"
+	+ "HXasIijFqA0+huwNNXk99CmTvIkqUsIow8x5MSTp3gU9X6uW4QtbI/C5kiNCpWHm\n"
+	+ "Xr3H3mgz0kzYi4vCYxAf7fPM9qtNyo92feiDKLUhYNtfja/cM3BWzm/vpEwmz9v7\n"
+	+ "Mp9204GbQSontb4oDDJuZPN3i+XGHlRerlREg+ScjyQH0u1UjujmOervNeHCVaCs\n"
+	+ "/emamOzv/6HY9wAB3AsWxKDX5tbrAjHgm61cCC3UESoHDApyzE1OcaNu/AsLo4iq\n"
+	+ "NspCZTk9+TPXgQIDAQABo1AwTjAdBgNVHQ4EFgQU8v25fJq45gkYrS75spZ01W47\n"
+	+ "5oMwHwYDVR0jBBgwFoAU8v25fJq45gkYrS75spZ01W475oMwDAYDVR0TBAUwAwEB\n"
+	+ "/zANBgkqhkiG9w0BAQsFAAOCAQEAQ48yKSwoPEmc7kGvrRIqdnBuZMAaGBO72xOZ\n"
+	+ "S4jQl2OPMYxGRWW7Km21L9vocDim9GYSmaKNm8uGv/E4XWpgyY609Vai7RdA1qEJ\n"
+	+ "JuW2KsmraxstfdUiXwYDPS4AC3hLtnW473mxyfa0BdzJMeO+/ZYrDppJhk1kvbHa\n"
+	+ "usgBI7vgzTbckq9+vNoGdrvzHSHNbxW5xKMO7thqgoUxijShSojJ/AXARtB6hBbG\n"
+	+ "EwOuXHe7Jop0nLzQe46jPgn9q+Fe2gYv26EskUXZAAoDKMZTYgjnfuXinZMKpQ7N\n"
+	+ "vUerIa96ds41U3YszcracJrHcz+qVtyWcyjAnGg6YIPIeSo/cw==\n"
+	+ "-----END CERTIFICATE-----\n";
+
+    @Test
+    public void testGoodKeyAndGoodCertificate() throws KeyManagementException, KeyStoreException {
+		String generatorAlias = DynamicKeystoreGenerator.KEYSTORE_ALIAS;
+
+		byte[] decodedPrivateKey = DynamicKeystoreGenerator.getDecodedPrivateKey(GOOD_TESTABLE_PRIVATE_KEY);
```

Comment:
Indentation issue.  Make sure to use spaces instead of tabs. Applies to all changes.